config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90

```diff
@@ -62,6 +62,8 @@ module MOM_ocean_model_nuopc
 use MOM_surface_forcing_nuopc, only : convert_IOB_to_forces, ice_ocn_bnd_type_chksum
 use MOM_surface_forcing_nuopc, only : ice_ocean_boundary_type, surface_forcing_CS
 use MOM_surface_forcing_nuopc, only : forcing_save_restart
+use MOM_domains,               only : root_PE,num_PEs
+use MOM_coms,                  only : Get_PElist
```

Comment:
Looks like these imports were added for debugging and aren't necessary anymore.

Reply:
They were left over from when I was calling stochastic physics from the cap.  I will remove.